core/src/main/java/org/apache/calcite/sql/SqlUtil.java

```diff
@@ -1069,6 +1069,20 @@ private void visitChild(SqlNode node) {
       return check(type);
     }
   }
+
+  /**
+   * Whether the selectItem is a table function node in the select.
+   * eg. "select table_func(1) as (f0,f1)"
+   * @param selectItem select item
+   * @return true is this selectItem is a table function call
+   */
```

Comment:
It should be `true if this....` instead of `true is this...`. Right?

Reply:
Thanks for correct me.